src/msw/printdlg.cpp

```diff
@@ -468,7 +474,7 @@ void wxWindowsPrintNativeData::InitializeDevMode(const wxString& printerName, Wi
         }
     }
 
-    if ( !m_devMode )
+    if ( !m_devMode && printerName.IsEmpty())
```

Comment:
Another minor style fix:
```suggestion
    if ( !m_devMode && printerName.IsEmpty() )
```